File-Formats/PDF/Working-with-OCR/aspnet-mvc.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Perform OCR on a PDF document in ASP.NET MVC | Syncfusion
+description: Learn how to efficiently perform OCR on a PDF document in ASP.NET MVC with easy steps using Syncfusion .NET library.
```

Comment:
Learn how to perform OCR on scanned PDF documents and images with different tesseract version in ASP.NET MVC using Syncfusion .NET OCR library.

Reply:
changed description